cli/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -566,14 +548,14 @@ pub mod test_helpers {
 
     use super::*;
 
-    pub fn new_path_environment(flox: &Flox, contents: &str) -> PathEnvironment {
+    pub fn new_path_environment(flox: &Flox, manifest: &RawManifest) -> PathEnvironment {
```

Comment:
suggestion: leave this accepting a `&str`. That will fix the broken tests, and I think it's easier to write tests when you can just pass a string

Reply:
Currently the `write_new` method is explicitly documented to write _anything_ entirely unverified.
IMO the more correct implementation would require validated inputs (e.g. CanonicalPath and RawManifest). But to a degree, `init` is already doing those checks.
It might be clearer to rather call the function `write_new_unchecked`, so its more clear why it takes lower level arguments.